test/examples/src.latex_simple/index.md

```diff
@@ -28,3 +28,26 @@ julia> 127 % Int8
 ```julia-repl sayhello2
 julia> function foo end;
 ```
+
+## Issue 1401
+
+1. The output is a latex tabular environment
+
+```@example
+using DataFrames
+DataFrame(i=1:3, y='A':'C')
+```
+
+2. The paragraph itself is a latex tabular environment
+
+\begin{tabular}{r|ccc}
+        & i & y & z\\
+        \hline
+        & Int64 & Char & Int64\\
+        \hline
+        1 & 1 & 'A' & 5 \\
+        2 & 2 & 'B' & 6 \\
+        3 & 3 & 'C' & 7 \\
+        4 & 4 & 'D' & 8 \\
+\end{tabular}
```

Comment:
Do you intend this to render as text or a table? Because it _should_ render as just plain text. If you indeed want to include raw LaTeX, you'd need to use [the at-raw block](https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@raw-format-block).